app/config/services/platform.xml

```diff
@@ -217,6 +221,16 @@
             <tag name="kernel.event_subscriber"/>
         </service>
 
+        <service id="app.citizen_project.cover_generator" class="AppBundle\MediaGenerator\Image\CitizenProjectCoverGenerator">
+            <argument type="service" id="knp_snappy.image"/>
+            <argument type="service" id="templating"/>
+        </service>
+
+        <service id="app.citizen_project.tract_generator" class="AppBundle\MediaGenerator\Pdf\CitizenProjectTractGenerator">
```

Comment:
If you could use the classname as id it would be better (for the future for autowiring etc.)